Makefile

```diff
@@ -30,33 +28,37 @@ test-filenames:
 	echo $(SHARED_LIB_DST)
 	echo $(SHARED_LIB_SRC)
 
-all: build test
-
-build: build-rust build-go
-
-build-rust: build-rust-release
+.PHONY: build
+build:
+	make build-libwasmvm
+	make build-go
```

Comment:
In the dependency notation we had before they may run in parallel (`make build -j 2` would run both jobs at the same time)